### PR TITLE
[PCFG-206] Creditor institution - station details

### PR DESCRIPTION
**Pull request by aomegax**
#### List of Changes
- updated creditor institution - station relationship details

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/main/java/it/pagopa/pagopa/apiconfig/model/creditorinstitution/StationCreditorInstitutions.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+@Builder(toBuilder = true)
+@NoArgsConstructor
+@AllArgsConstructor
+@ToString
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class StationCreditorInstitutions {
+    @JsonProperty("creditor_institutions")
+    @Schema(required = true)
+    @NotNull
```
Comment (jacopocarlini) [inline src/main/java/it/pagopa/pagopa/apiconfig/model/creditorinstitution/StationCreditorInstitutions.java:27]: can it be empty? If not use @NotEmpty

Comment (jacopocarlini) [inline src/main/java/it/pagopa/pagopa/apiconfig/model/creditorinstitution/StationCreditorInstitutions.java:27]: can it be empty? If not use @NotEmpty

**Review by jacopocarlini — APPROVED** [review]
lgtm 👍

File: src/main/java/it/pagopa/pagopa/apiconfig/model/creditorinstitution/StationCreditorInstitutions.java (hunk 1 above)
Comment (aomegax) [reply src/main/java/it/pagopa/pagopa/apiconfig/model/creditorinstitution/StationCreditorInstitutions.java:27]: it could be empty list.

**Pull request merged**